src/localizations/en.json

```diff
@@ -139,7 +139,9 @@
     "own_orgnumber_delegation_heading": "You can't delegate to your own organisation number",
     "own_orgnumber_delegation_paragraph": "Search for another organisation number to continue.",
     "no_orgs_chosen": "No chosen organisations",
-    "insufficient_authentication_level": "You don't have secure enough login to delegate the service. Please log in with one of the methods below:"
+    "insufficient_authentication_level": "You don't have secure enough login to delegate the service. Please log in with one of the methods below:",
+    "revoke_delegation_failed": "Failed to delete",
+    "failed_to_revoke": "{{apiId}} {{orgNumber}}"
```

Comment:
Why keep this as a localized text key and not simply print the variables directly?

Reply:
A good point.